spec/index.html

```diff
@@ -588,6 +588,43 @@ <h2>Namespace Prefixes</h2>
     For convenience, normative statements for documents are often phrased
     as statements on the properties of the document.
   </p>
+
+  <section id="v11">
+    <h2>JSON-LD Version</h2>
+
+    <p>YAML-LD supports JSON-LD versions starting from 1.1 [[JSON-LD11]].</p>
+  </section>
+
+  <section id="testing">
+    <h2>Test Suites</h2>
+
+    <p>To be conformant, an implementation MUST satisfy all test cases from the following test suites:</p>
+
+    <ul>
+      <li>
+        <a href="https://json-ld.github.io/yaml-ld/tests/">YAML-LD tests</a>;
+      </li>
+      <li>
+        <a href="https://w3c.github.io/json-ld-api/tests/">JSON-LD API tests</a>;
+      </li>
+      <li>
+        <a href="https://w3c.github.io/json-ld-framing/tests/">JSON-LD Framing tests</a>,
+      </li>
+    </ul>
+
+    <p>...with the exclusion of the test cases where:</p>
+
+    <ul>
+      <li>
+        <a href="#dom-jsonldoptions-processingmode">processingMode</a> option is provided and set to `json-ld-1.0`.
```

Comment:
```suggestion
        <a href="#dom-jsonldoptions-processingmode">processingMode</a> option is provided and set to `json-ld-1.1`.
```

Reply:
Hmm, the intention here is to specify which test cases to ignore, these should be 1.0 test cases... or am I confusing something? @gkellogg

Reply:
Indeed, I didn't read closely enough. I withdraw my suggestion.